go-runtime/compile/schema_test.go

```diff
@@ -398,46 +398,48 @@ func TestErrorReporting(t *testing.T) {
+		`14:18-52: duplicate config declaration at 13:18-52`,
+		`17:18-52: duplicate secret declaration at 16:18-52`,
+		`20:14-44: duplicate database declaration at 19:14-44`,
+		`23:2-10: unsupported type "error" for field "BadParam"`,
+		`26:2-17: unsupported type "uint64" for field "AnotherBadParam"`,
+		`29:3-3: unexpected token "export" (expected Directive)`,
+		`35:36-39: unsupported request type "ftl/failing.Request"`,
+		`35:50-50: unsupported response type "ftl/failing.Response"`,
+		`36:16-29: call first argument must be a function but is an unresolved reference to lib.OtherFunc`,
+		`36:16-29: call first argument must be a function in an ftl module`,
+		`37:2-46: call must have exactly three arguments`,
+		`38:16-25: call first argument must be a function in an ftl module`,
+		`43:1-2: must have at most two parameters (context.Context, struct)`,
+		`43:69-69: unsupported response type "ftl/failing.Response"`,
+		`48:22-27: first parameter must be of type context.Context but is ftl/failing.Request`,
+		`48:37-43: second parameter must be a struct but is string`,
+		`48:53-53: unsupported response type "ftl/failing.Response"`,
+		`53:43-47: second parameter must not be ftl.Unit`,
+		`53:59-59: unsupported response type "ftl/failing.Response"`,
+		`58:1-2: first parameter must be context.Context`,
+		`58:18-18: unsupported response type "ftl/failing.Response"`,
+		`63:1-2: must have at most two results (struct, error)`,
+		`63:41-44: unsupported request type "ftl/failing.Request"`,
+		`68:1-2: must at least return an error`,
+		`68:36-39: unsupported request type "ftl/failing.Request"`,
+		`72:35-38: unsupported request type "ftl/failing.Request"`,
+		`72:48-48: must return an error but is ftl/failing.Response`,
+		`77:41-44: unsupported request type "ftl/failing.Request"`,
+		`77:55-55: first result must be a struct but is string`,
+		`77:63-63: must return an error but is string`,
+		`77:63-63: second result must not be ftl.Unit`,
+		`84:1-1: duplicate verb name "WrongResponse"`,
+		`90:2-12: struct field unexported must be exported by starting with an uppercase letter`,
+		`102:2-24: cannot attach enum value to BadValueEnum because it is a variant of type enum TypeEnum, not a value enum`,
+		`109:2-41: cannot attach enum value to BadValueEnumOrderDoesntMatter because it is a variant of type enum TypeEnum, not a value enum`,
+		`118:1-26: parent enum "ExportedTypeEnum" is exported, but directive "ftl:data" on "PrivateData" is not: all variants of exported enums that have a directive must be explicitly exported as well`,
+		`122:21-60: config and secret names must be valid identifiers`,
+		`128:1-26: only one directive expected for type alias`,
+		`144:1-35: type can not be a variant of more than 1 type enums (TypeEnum1, TypeEnum2)`,
+		`152:2-2: unsupported external type ftl/failing/child.ChildStruct`,
+		`152:2-6: unsupported type "ftl/failing/child.ChildStruct" for field "Data"`,
```

Comment:
the 2 errors for line 152 are what are actually being surfaced with the new test change